### DISCUSSION TITLE
我们应该学习二舅勇敢地接收生活吗？

### DISCUSSION THREAD

**Discussion by qqqq242b** [WeNeedHome/SummaryOfLoanSuspension#1032] [General]
https://www.bilibili.com/video/BV1dr4y1L7cN

**Comment by frshgrass** [comment]
二舅能这样做是因为：
      1，超高智商，学东西快，有很多技能借以糊口和才受村民尊重（是因为你有用才能这样，在农村生活都知道，农村很丛林法则的，你没本事，家族大人多会欺负你到死）。这种智商天赋我说他千中无一不过分吧。

2，二舅面对苦难，能感受到他的坚强意志和乐观。但是乐观努力意志坚强这类情商也是与生俱来的天赋（其实努力和意志算不算天赋，之前知乎讨论过，很多人现身说法就是，部分人觉得坚持和努力并不辛苦，他们天生就这样，也喜欢这样的状态）。那这种情商我说他万中无一不过分吧

所以呢，二舅在情商和智商的双重天赋下，在面对苦难的漫天炮火，天赋自带的高情商和智商犹如给二舅穿上了厚厚的防弹盔甲，即使这样也是过着普通人的生活。

那我们呢 智商和情商天赋一般的人  明知道有坏人在制造苦难，你觉得你的情商和智商比得过二舅吗 身披草甲平凡的我们面对炮火直接几下就无了。（我想过我要是遇到二舅这样的苦难，我的智商和心性 要不抑郁自杀 要不变成村口树先生 疯掉）

二舅的智商和情商天赋所在万中无一，也选择不娶妻孑然一身，才可以在厚厚的铠甲下迎面炮火。

但是世界上还有很多生下来就像我们这样读书永远是中流或者拼熬夜拼辛苦，然后上有老下有小各方面压力也大，心思沉重敏感的平凡人 。我们如果独自面对炮火直接无了。

 我们的孩子大部分也是情商和智商的普通人，我们还自己把最好的一面展现，然后教育要善良让孩子单纯得像天使一样。你敢让我们的孩子面对这样的炮火吗？

应该团结起来 砸烂苦难的制造者  我们要接受自己只是普通人  普通人要在没有恶人和炮火世界开心得生活。

万中无一的，集天赋决定的情商和智商的，幸存者偏差。我们只是天赋一般的普通人，如果信了这套话术包装的鸡汤，赤身裸体的接受苦难的炮火，你将慢慢的血肉横飞无掉了。

无非是改头换面的于丹讲论语，话术更自然而已（呵呵 也不知道 火了之后 二舅的残疾证还能不能办下来）

苦难要咬紧牙关 但从来不值得歌颂（失职的地方政府 我也不知道 他哪来的脸宣传这些）

**Reply by zxcs001** [reply]
你可以学习，但是我不学，完毕……

**Comment by mathLjtao** [comment]
加油，侠之大者们

**Comment by shawn-yee** [comment]
你问二舅，来世投胎还来华夏大地吗?

**Comment by Accelerator6666** [comment]
毒鸡汤。。。千万别往心里去

**Comment by xiejun946** [comment]
是自己的权益就要自己争取，谁也别想抢走

**Comment by redapple2021** [comment]
这不叫接受生活，这是像牲口一样活着！
自己的权益自己捍卫

**Comment by xujiyou** [comment]
你还能去死不成

**Comment by chosai** [comment]
对对对，紫蜡烛，不给老爷交贷款就是不爱国，快爱国的好好生活，不要停贷

**Comment by faceAndfaceandface** [comment]
劝导别人要接受苦难，自己却一边岁月静好，这......

**Comment by itsoursduty** [comment]
世界上最无耻，最阴险，最歹毒的赞美，就是用穷人的艰辛和苦难，当作励志故事愚弄底层人
